packages/grafana-ui/src/components/Toggletip/Toggletip.tsx

```diff
@@ -28,6 +29,8 @@ export interface ToggletipProps {
   footer?: JSX.Element | string;
   /** The UI control users interact with to display toggletips */
   children: JSX.Element;
+  /** The class name to be applied to the toggletip's container */
+  containerClassName?: string;
```

Comment:
We are avoiding adding `className` props so that we don't encourage developers to override most of the styling. But I agree that the max-width that is set here does not make sense for your use case.

I would propose a more explicit prop here, either `maxWidth` directly or a boolean `fitContentWidth`.

Reply:
done!